src/Stackctl/RemovedStack.hs

```diff
@@ -0,0 +1,49 @@
+module Stackctl.RemovedStack
+  ( RemovedStack(..)
```

Comment:
This was from an older version of this. I'm not using it anymore and will remove.

Reply:
did you mean you would remove this as part of this PR, or after it merges?

Reply:
As part of this PR.